src/vmm/src/devices/virtio/balloon/device.rs

```diff
@@ -285,7 +286,7 @@ impl Balloon {
     pub(crate) fn process_inflate(&mut self) -> Result<(), BalloonError> {
         // This is safe since we checked in the event handler that the device is activated.
         let mem = self.device_state.mem().unwrap();
-        METRICS.balloon.inflate_count.inc();
+        BALLOON_METRICS.inflate_count.inc();
```

Comment:
Same comment as before.

Reply:
I agree and would probably lift this above being a `nit` on my side: Its now `vmm::devices::balloon::balloon_metrics::BALLOON_METRICS`. That's too much `balloon` in the path imo.

Reply:
Yes, I was thinking about not making this a `nit` either. Now that I see the name space, maybe it's worth doing this:

`vmm::devices::balloon::metrics::METRICS` (so renaming the `balloon_metrics` module as well. However, I'm happy with this coming as a follow-up as well.